packages/integration-karma/test/template/attribute-style/index.spec.js

```diff
@@ -11,6 +11,7 @@ describe('static style attribute', () => {
         const target = elm.shadowRoot.querySelector('div');
         expect(target.style.position).toBe('absolute');
         expect(target.style.top).toBe('10px');
+        expect(target.style.getPropertyValue('--custom-property')).toBe('blue');
```

Comment:
This will certainly fail on IE11.

Reply:
I'm curious to see your strategy to add tests that behave differently per browser.